src/uct/ib/rc/base/rc_iface.c

```diff
@@ -14,7 +14,7 @@
 
 
 ucs_config_field_t uct_rc_iface_config_table[] = {
-  {"IB_", "RX_INLINE=64;RX_QUEUE_LEN=4095", NULL,
+  {"IB_", "RX_INLINE=64;RX_QUEUE_LEN=256", NULL,
```

Comment:
it can have negative effect on many-to-one performance, also lower than FC window
@brminich pls comment